examples/generative_models/jtnn/README.md

```diff
@@ -31,10 +56,25 @@ encoded nodes(atoms) and edges(bonds), and other information for model to use.
 ## Usage
 
 ### Training
-
-To start training, use `python train.py`. By default, the script will use ZINC dataset
+####ZINC
+For ZINC dataset, to start training, use `python train.py`. By default, the script will use ZINC dataset
  with preprocessed vocabulary, and save model checkpoint periodically in the current working directory. 
-
+ 
+####GuacaMol
+For GuacaMol dataset, you need to process the training dataset to get vocabulary first with
+```
+python vocab.py -d $DATASET -v  $VOCABULARY
+```
+`$DATASET` specifies the filename of the training-set.  
+`$VOCABULARY` specifies the filename to save the processed vocabulary.  
```

Comment:
Since I've uploaded the vocabulary file of GuacaMol to an S3 bucket, I don't think users need to re-process the vocabulary file. Have you tried using the vocabulary file from S3?